src/riscv-integration.adoc

```diff
@@ -325,24 +325,31 @@ NOTE: *CHERI v9 Note:* CSpecialRW is removed. Its role is assumed by
 endif::[]
 
 All CSR instructions atomically read-modify-write a single CSR. If the CSR
-accessed is of capability size then the
-capability's tag, metadata and address are all accessed atomically.
-
-When the <<CSRRW>> instruction is accessing a capability width CSR, then the source
-and destination operands are *c* registers and it atomically swaps the values in the
-whole CSR with the CLEN width register operand.
-
-There are special rules for updating specific CLEN-wide CSRs as shown in <<extended_CSR_writing>>.
-
-{non-csrrw-and} specify *x* registers and so only access the address field
-of the capability when specifying a capability CSR such as <<mtvecc>>.
-They calculate the final address using the standard
-RISC-V behaviour (set bits, clear bits etc.) and that final address is
-updated in the capability. The update typically uses the semantics of a <<CSETADDR>>
-instruction which clears the tag if the capability is sealed, or if the
-updated address is not representable. <<extended_CSR_writing>> shows the
+accessed is of capability size then the capability's tag, metadata and address
+are all accessed atomically.
```

Comment:
please don't reformat - or can it be excluded from the diff somehow? there's no change here